game/src/debug/debug_plugin.rs

```diff
@@ -0,0 +1,15 @@
+use bevy::prelude::*;
+
+pub struct DebugPlugin;
+
+//add the system(s) you wish to use to see debug info
+impl Plugin for DebugPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_systems(Update, print_test);
```

Comment:
"Debugger active" message should only show in the `Startup` schedule.

Reply:
Updated